### PR TITLE
Remove duplicate error code table of meshery-server in Error Code Ref…

### PR DESCRIPTION
**Pull request by 102vansh**
**Title:**
[UI] [Bug] Remove duplicate error code table of meshery-server in Error Code Reference #11942

**Description:**
This PR addresses the issue #11942 where the error code table for `meshery-server` was duplicated in the Error Code Reference page. The duplicate table has been removed, and the page now correctly displays a single table for `meshery-server` error codes.

**Notes for Reviewers:**
Please review the changes to ensure that the duplicate error code table has been removed and that the remaining content is accurate and correctly displayed.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 95345efd4e0c84c4af1cb458015ee44e9a36f2a1 at: https://66e04b9eb04f413599ed743e--meshery-docs-preview.netlify.app

File: docs/pages/reference/error-codes.md [hunk 1]
```diff
@@ -75,16 +73,30 @@ td {
 
 ## Error Codes and Troubleshooting
 
-Meshery and its components use a common framework (defined within MeshKit) to generate and document an event with a unique error code identifier as the combination of `[component type]-[component name]-[event moniker]-[numeric code]`. Each error code identifies the source component for the error and a standard set of information to describe the error and provide helpful details for troubleshooting the situation surrounding the specific error.
+Meshery and its components use a common framework (defined within MeshKit) to generate and document an event with a unique error code identifier as the combination of `[component type]-[component name]-[event moniker]-[numeric code]`. Each error code identifies the source component for the error and provides details to help troubleshoot the specific error.
 
-{% include alert.html type="info" title="Error codes are combination of component type, component name, event moniker and numberic code" content="Error codes are a hyphenated collection of details that include:
+{% include alert.html type="info" title="Error codes format" content="Error codes are a hyphenated collection of details:
```
Comment (leecalcote) [inline docs/pages/reference/error-codes.md:78]: Please revert this change.

File: docs/pages/reference/error-codes.md [hunk 2]
```diff
@@ -75,16 +73,30 @@ td {
 
 ## Error Codes and Troubleshooting
 
-Meshery and its components use a common framework (defined within MeshKit) to generate and document an event with a unique error code identifier as the combination of `[component type]-[component name]-[event moniker]-[numeric code]`. Each error code identifies the source component for the error and a standard set of information to describe the error and provide helpful details for troubleshooting the situation surrounding the specific error.
+Meshery and its components use a common framework (defined within MeshKit) to generate and document an event with a unique error code identifier as the combination of `[component type]-[component name]-[event moniker]-[numeric code]`. Each error code identifies the source component for the error and provides details to help troubleshoot the specific error.
```
Comment (leecalcote) [inline docs/pages/reference/error-codes.md:76]: Please revert this change.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks for refactoring, @102vansh 👏 

It looks like we might be missing an opening or a closing `if` / `endif` - https://github.com/meshery/meshery/actions/runs/10799459830/job/29955241832?pr=11951#step:4:354. Will you double-check?

**Comment by Ashparshp** [comment]
Thank you @102vansh
Let's discuss this PR on Meshery development call today. Adding it as an agenda item to the meeting minutes.

Timing: 9:00 AM CT
Meeting minutes: [Google Docs](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit)
Meeting link: [meet.layer5.io/meshery](https://meet.layer5.io/meshery)

**Comment by sudhanshutech** [comment]
@102vansh build is failing. Have you fix the endif syntax which is coming in build error and also mentioned by @leecalcote

**Pull request closed without merging**